includes/class-dispatcher.php

```diff
@@ -87,6 +88,42 @@ public function _execute_view( $template ) {
 		return $template;
 	}
 
+    /**
+     * Redirect from draft listings to the main directory page.
+     *
+     * @since x.x.x
```

Comment:
This should be x.x so it gets replaced correctly. Here's where the replacement is handled:
https://github.com/Strategy11/business-directory-plugin/blob/master/Gruntfile.js#L210